tests/support/psci/Sample.purs

```diff
@@ -0,0 +1,13 @@
+module Sample where
+
+import Prelude
+import Control.Monad.Eff (Eff)
+import Control.Monad.Eff.Console (CONSOLE, log)
+
+main :: forall e. Eff (console :: CONSOLE | e) Unit
+main = do
+  log "Hello sailor!"
+
+another :: forall e. Eff (console :: CONSOLE | e) Unit
```

Comment:
Perhaps it's simpler to just append these lines onto the file?

Ideally, I'd prefer to do this by mocking file IO. Could you please see if that's possible?

Reply:
Completely agree, will have a bit more of a dig. Thanks for taking a look.